packages/server/graphql/public/typeDefs/updateSAML.graphql

```diff
@@ -0,0 +1,28 @@
+extend type Mutation {
+  """
+  Describe the mutation here
+  """
+  updateSAML(
+    """
+    The organization id
+    """
+    orgId: ID!
+
+    """
+    A big chunk of XML data containing the redirect URL and X.509 certificate
+    """
+    metadata: String
+  ): UpdateSAMLPayload!
+}
+
+"""
+Return value for updateSAML, which could be an error
+"""
+union UpdateSAMLPayload = ErrorPayload | UpdateSAMLSuccess
+
+type UpdateSAMLSuccess {
+  """
+  Describe the first return field here
+  """
+  success: Boolean!
```

Comment:
@Dschoordsch I'm not 100% sure how this is going to work, do we need to return newly created/updated object id here and be able to resolve it? I'm asking as I'm not sure if frontend needs more data than `true/false`

Reply:
@BartoszJarocki, here is a  [draft](https://github.com/ParabolInc/parabol/pull/7788) for frontend, where we use this `mutation`  for enabling and disabling SSO. we have opted to perform a refetch to inform future decisions based on the query result.